geopandas/tools/geocoding.py

```diff
@@ -1,5 +1,6 @@
+from __future__ import annotations
```

Comment:
Can you remove this and all type hints from the code? We currenly don't use them and I'd like to avoid introducing inconsistency.